doc/doxygen/src/flashing.md

```diff
@@ -37,7 +37,7 @@ listed as supported for MCU family `BAR`.
 This table will use the value to pass to `PROGRAMMER=<value>` as title, rather
 than the official spelling of the programmer.
 
-MCU Family     | `adafruit-nrfutil` | `avrdude` | `bmp` | `bossa` | `cc2538-bsl`| `cpy2remed` | `dfu-util` | `edbg` | `elf2uf2` | `esptool` | `goodfet` | `jlink` | `lpc2k_pgm` | `mspdebug` | `nrfutil` | `openocd` | `pic32prog` | `pyocd` | `robotis-loader` | `stm32flash` | `stm32loader` | `uf2conv` | `uniflash`
+MCU Family     | `adafruit-nrfutil` | `avrdude` | `bmp` | `bossa` | `cc2538-bsl`| `cpy2remed` | `dfu-util` | `edbg` | `elf2uf2` | `esptool` | `goodfet` | `jlink` | `lpc2k_pgm` | `mspdebug` | `nrfutil` | `openocd` | `pyocd` | `robotis-loader` | `stm32flash` | `stm32loader` | `uf2conv` | `uniflash`
```

Comment:
Removing one column caption would need to remove all of the column contents.

Maybe it really was a poor idea to include single-platform flash tools in this matrix. May I should have created a matrix for multi-platform flash tools and listed the single-platform flashers separately.

Reply:
I split out the single-purpose tools in PR https://github.com/RIOT-OS/RIOT/pull/18577